src/utils/calendar.ts

```diff
@@ -22,7 +22,11 @@ export const createCalendar = ({
       organizer: `${event.host || "Unknown"} <undefined>`, // string format: "name <email>". email cannot be blank
       description: event.description,
       summary: event.name,
-      url: getFullVenueInsideUrl(event.venueId),
+      url: generateAttendeeInsideSpaceUrl({
+        worldSlug: event.worldSlug,
+        spaceSlug: event.venueSlug,
```

Comment:
Slugs should be read from their respective objects, not an event one that might have an outdated value for them